### PR TITLE
Update create-test-subset 

### PR DESCRIPTION
**Pull request by vivbak**
Pulls participant, family & pedigree information to test project.

### CODE REVIEW

**Comment by vivbak** [comment]
Depends on https://github.com/populationgenomics/sample-metadata/pull/150

File: scripts/create_test_subset.py [hunk 1]
```diff
@@ -187,15 +192,67 @@ def main(
...
+    tmp_family_tsv = 'tmp_families.tsv'
+    family_tsv_headers = ['Family ID', 'Description', 'Coded Phenotype', 'Display Name']
+    # Work-around as import_families takes a file.
+    with open(tmp_family_tsv, 'wt') as tmp_families:
+        tsv_writer = csv.writer(tmp_families, delimiter='\t')
+        tsv_writer.writerow(family_tsv_headers)
+        for family in families:
+            del family['id']
+            del family['project']
+            tsv_writer.writerow(list(family.values()))
```
Comment (illusional) [inline scripts/create_test_subset.py:233]: I'd probably be more explicit with the order you write this in. Trusting the response from the SM server to be in the order with the correct keys is a bit dangerous. I'd probably prefer something that explicitly builds the keys:

```python
tsv_writer.writerow([
    family['external_id'],
    family['description'] or '',
# .... etc
```

Comment (vivbak) [reply scripts/create_test_subset.py:233]: ah, good point -- thanks! Will fix :)

**Review by vladsaveliev — APPROVED** [review]
Looks good! Thanks Vivian :)

**Pull request merged**